librz/core/cmd/cmd_print.c

```diff
@@ -5233,14 +5018,9 @@ RZ_IPI int rz_cmd_print(void *data, const char *input) {
 			}
 			break;
 		}
-		case 'd': // "pId" is the same as pDi
-			if (l) {
-				rz_core_disasm_pdi(core, 0, l, 0);
-			}
-			break;
```

Comment:
what is replacing `pDi`?

Reply:
`pId` is an alias for `pDi`, now they are all assigned to `pDq`
Similarly, `pid` | `pdi` -> `pdq`